Update airtame.sh

hwlans opened the pull request:
**Have you confirmed this pull request is not a duplicate?**
Yes

**Is this pull request creating or modifying a label in the fragments/labels folder, and not Installomator.sh itself?**
Yes

**Did you use [our editorconfig file](https://github.com/Installomator/Installomator/wiki/Contributing-to-Installomator)?**
Yes

**Additional context** Add any other context about the label or fix here.
`downloadURL` changed.
I had to remove `appNewVersion`; the new download url is keeping us in the dark.

**Installomator log** At the bottom of this pull request, provide a log of a label run by running Installomator in Terminal and saving the output. `DEBUG=1` can be enabled but **do not enable [Debug logging level](https://github.com/Installomator/Installomator/wiki/Configuration-and-Variables#logging-level) and please format the log [using a code block](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks)!**

Please identify any issues fixed by your pull request by including the issue number. (Example: "Fixes #XXXX")
Fixes #2463 

````
./assemble.sh airtame
2025-07-06 22:38:57 : INFO  : airtame : Total items in argumentsArray: 0
2025-07-06 22:38:57 : INFO  : airtame : argumentsArray:
2025-07-06 22:38:57 : REQ   : airtame : ################## Start Installomator v. 10.9beta, date 2025-07-06
2025-07-06 22:38:57 : INFO  : airtame : ################## Version: 10.9beta
2025-07-06 22:38:57 : INFO  : airtame : ################## Date: 2025-07-06
2025-07-06 22:38:57 : INFO  : airtame : ################## airtame
2025-07-06 22:38:57 : DEBUG : airtame : DEBUG mode 1 enabled.
2025-07-06 22:38:58 : INFO  : airtame : Reading arguments again:
2025-07-06 22:38:58 : DEBUG : airtame : name=Airtame
2025-07-06 22:38:58 : DEBUG : airtame : appName=
2025-07-06 22:38:58 : DEBUG : airtame : type=dmg
2025-07-06 22:38:58 : DEBUG : airtame : archiveName=
2025-07-06 22:38:58 : DEBUG : airtame : downloadURL=https://downloads-website.airtame.com/get.php?platform=mac
2025-07-06 22:38:58 : DEBUG : airtame : curlOptions=
2025-07-06 22:38:58 : DEBUG : airtame : appNewVersion=
2025-07-06 22:38:58 : DEBUG : airtame : appCustomVersion function: Not defined
2025-07-06 22:38:58 : DEBUG : airtame : versionKey=CFBundleShortVersionString
2025-07-06 22:38:58 : DEBUG : airtame : packageID=
2025-07-06 22:38:58 : DEBUG : airtame : pkgName=
2025-07-06 22:38:58 : DEBUG : airtame : choiceChangesXML=
2025-07-06 22:38:58 : DEBUG : airtame : expectedTeamID=4TPSP88HN2
2025-07-06 22:38:58 : DEBUG : airtame : blockingProcesses=
2025-07-06 22:38:58 : DEBUG : airtame : installerTool=
2025-07-06 22:38:58 : DEBUG : airtame : CLIInstaller=
2025-07-06 22:38:58 : DEBUG : airtame : CLIArguments=
2025-07-06 22:38:58 : DEBUG : airtame : updateTool=
2025-07-06 22:38:58 : DEBUG : airtame : updateToolArguments=
2025-07-06 22:38:58 : DEBUG : airtame : updateToolRunAsCurrentUser=
2025-07-06 22:38:58 : INFO  : airtame : BLOCKING_PROCESS_ACTION=tell_user
2025-07-06 22:38:58 : INFO  : airtame : NOTIFY=success
2025-07-06 22:38:58 : INFO  : airtame : LOGGING=DEBUG
2025-07-06 22:38:58 : INFO  : airtame : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-07-06 22:38:58 : INFO  : airtame : Label type: dmg
2025-07-06 22:38:58 : INFO  : airtame : archiveName: Airtame.dmg
2025-07-06 22:38:58 : INFO  : airtame : no blocking processes defined, using Airtame as default
2025-07-06 22:38:58 : DEBUG : airtame : Changing directory to /Users/h.lans/Documents/GitHub/Installomator/build
2025-07-06 22:38:58 : INFO  : airtame : name: Airtame, appName: Airtame.app
2025-07-06 22:38:58 : WARN  : airtame : No previous app found
2025-07-06 22:38:58 : WARN  : airtame : could not find Airtame.app
2025-07-06 22:38:58 : INFO  : airtame : appversion:
2025-07-06 22:38:58 : INFO  : airtame : Latest version not specified.
2025-07-06 22:38:58 : REQ   : airtame : Downloading https://downloads-website.airtame.com/get.php?platform=mac to Airtame.dmg
2025-07-06 22:38:58 : DEBUG : airtame : No Dialog connection, just download
2025-07-06 22:39:09 : INFO  : airtame : Downloaded Airtame.dmg – Type is  zlib compressed data – SHA is cfa26adc44c23ceb37504721b0016d08cc813ba6 – Size is 131596 kB
2025-07-06 22:39:09 : DEBUG : airtame : DEBUG mode 1, not checking for blocking processes
2025-07-06 22:39:09 : REQ   : airtame : Installing Airtame
2025-07-06 22:39:09 : INFO  : airtame : Mounting /Users/h.lans/Documents/GitHub/Installomator/build/Airtame.dmg
2025-07-06 22:39:14 : DEBUG : airtame : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified CRC32 $0E1587F0
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified CRC32 $A0848190
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified CRC32 $5D8799FA
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified CRC32 $7C147529
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified CRC32 $5D8799FA
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified CRC32 $CABB07C9
verified CRC32 $D353CFAA
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/Airtame 4.13.1

2025-07-06 22:39:14 : INFO  : airtame : Mounted: /Volumes/Airtame 4.13.1
2025-07-06 22:39:14 : INFO  : airtame : Verifying: /Volumes/Airtame 4.13.1/Airtame.app
2025-07-06 22:39:14 : DEBUG : airtame : App size: 268M	/Volumes/Airtame 4.13.1/Airtame.app
2025-07-06 22:39:16 : DEBUG : airtame : Debugging enabled, App Verification output was:
/Volumes/Airtame 4.13.1/Airtame.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: AIRTAME APS (4TPSP88HN2)

2025-07-06 22:39:16 : INFO  : airtame : Team ID matching: 4TPSP88HN2 (expected: 4TPSP88HN2 )
2025-07-06 22:39:16 : INFO  : airtame : Installing Airtame version 4.13.1 on versionKey CFBundleShortVersionString.
2025-07-06 22:39:16 : INFO  : airtame : App has LSMinimumSystemVersion: 10.15
2025-07-06 22:39:16 : DEBUG : airtame : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-07-06 22:39:16 : INFO  : airtame : Finishing...
2025-07-06 22:39:19 : INFO  : airtame : name: Airtame, appName: Airtame.app
2025-07-06 22:39:19 : WARN  : airtame : No previous app found
2025-07-06 22:39:19 : WARN  : airtame : could not find Airtame.app
2025-07-06 22:39:19 : REQ   : airtame : Installed Airtame, version 4.13.1
2025-07-06 22:39:19 : INFO  : airtame : notifying
2025-07-06 22:39:19 : DEBUG : airtame : Unmounting /Volumes/Airtame 4.13.1
2025-07-06 22:39:19 : DEBUG : airtame : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-07-06 22:39:19 : DEBUG : airtame : DEBUG mode 1, not reopening anything
2025-07-06 22:39:19 : REQ   : airtame : All done!
2025-07-06 22:39:19 : REQ   : airtame : ################## End Installomator, exit code 0
````

hwlans committed:
Update airtame.sh

````
./assemble.sh airtame
2025-07-06 22:38:57 : INFO  : airtame : Total items in argumentsArray: 0
2025-07-06 22:38:57 : INFO  : airtame : argumentsArray:
2025-07-06 22:38:57 : REQ   : airtame : ################## Start Installomator v. 10.9beta, date 2025-07-06
2025-07-06 22:38:57 : INFO  : airtame : ################## Version: 10.9beta
2025-07-06 22:38:57 : INFO  : airtame : ################## Date: 2025-07-06
2025-07-06 22:38:57 : INFO  : airtame : ################## airtame
2025-07-06 22:38:57 : DEBUG : airtame : DEBUG mode 1 enabled.
2025-07-06 22:38:58 : INFO  : airtame : Reading arguments again:
2025-07-06 22:38:58 : DEBUG : airtame : name=Airtame
2025-07-06 22:38:58 : DEBUG : airtame : appName=
2025-07-06 22:38:58 : DEBUG : airtame : type=dmg
2025-07-06 22:38:58 : DEBUG : airtame : archiveName=
2025-07-06 22:38:58 : DEBUG : airtame : downloadURL=https://downloads-website.airtame.com/get.php?platform=mac
2025-07-06 22:38:58 : DEBUG : airtame : curlOptions=
2025-07-06 22:38:58 : DEBUG : airtame : appNewVersion=
2025-07-06 22:38:58 : DEBUG : airtame : appCustomVersion function: Not defined
2025-07-06 22:38:58 : DEBUG : airtame : versionKey=CFBundleShortVersionString
2025-07-06 22:38:58 : DEBUG : airtame : packageID=
2025-07-06 22:38:58 : DEBUG : airtame : pkgName=
2025-07-06 22:38:58 : DEBUG : airtame : choiceChangesXML=
2025-07-06 22:38:58 : DEBUG : airtame : expectedTeamID=4TPSP88HN2
2025-07-06 22:38:58 : DEBUG : airtame : blockingProcesses=
2025-07-06 22:38:58 : DEBUG : airtame : installerTool=
2025-07-06 22:38:58 : DEBUG : airtame : CLIInstaller=
2025-07-06 22:38:58 : DEBUG : airtame : CLIArguments=
2025-07-06 22:38:58 : DEBUG : airtame : updateTool=
2025-07-06 22:38:58 : DEBUG : airtame : updateToolArguments=
2025-07-06 22:38:58 : DEBUG : airtame : updateToolRunAsCurrentUser=
2025-07-06 22:38:58 : INFO  : airtame : BLOCKING_PROCESS_ACTION=tell_user
2025-07-06 22:38:58 : INFO  : airtame : NOTIFY=success
2025-07-06 22:38:58 : INFO  : airtame : LOGGING=DEBUG
2025-07-06 22:38:58 : INFO  : airtame : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2025-07-06 22:38:58 : INFO  : airtame : Label type: dmg
2025-07-06 22:38:58 : INFO  : airtame : archiveName: Airtame.dmg
2025-07-06 22:38:58 : INFO  : airtame : no blocking processes defined, using Airtame as default
2025-07-06 22:38:58 : DEBUG : airtame : Changing directory to /Users/h.lans/Documents/GitHub/Installomator/build
2025-07-06 22:38:58 : INFO  : airtame : name: Airtame, appName: Airtame.app
2025-07-06 22:38:58 : WARN  : airtame : No previous app found
2025-07-06 22:38:58 : WARN  : airtame : could not find Airtame.app
2025-07-06 22:38:58 : INFO  : airtame : appversion:
2025-07-06 22:38:58 : INFO  : airtame : Latest version not specified.
2025-07-06 22:38:58 : REQ   : airtame : Downloading https://downloads-website.airtame.com/get.php?platform=mac to Airtame.dmg
2025-07-06 22:38:58 : DEBUG : airtame : No Dialog connection, just download
2025-07-06 22:39:09 : INFO  : airtame : Downloaded Airtame.dmg – Type is  zlib compressed data – SHA is cfa26adc44c23ceb37504721b0016d08cc813ba6 – Size is 131596 kB
2025-07-06 22:39:09 : DEBUG : airtame : DEBUG mode 1, not checking for blocking processes
2025-07-06 22:39:09 : REQ   : airtame : Installing Airtame
2025-07-06 22:39:09 : INFO  : airtame : Mounting /Users/h.lans/Documents/GitHub/Installomator/build/Airtame.dmg
2025-07-06 22:39:14 : DEBUG : airtame : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified CRC32 $0E1587F0
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified CRC32 $A0848190
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified CRC32 $5D8799FA
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified CRC32 $7C147529
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified CRC32 $5D8799FA
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified CRC32 $CABB07C9
verified CRC32 $D353CFAA
/dev/disk4          	GUID_partition_scheme
/dev/disk4s1        	Apple_HFS                      	/Volumes/Airtame 4.13.1

2025-07-06 22:39:14 : INFO  : airtame : Mounted: /Volumes/Airtame 4.13.1
2025-07-06 22:39:14 : INFO  : airtame : Verifying: /Volumes/Airtame 4.13.1/Airtame.app
2025-07-06 22:39:14 : DEBUG : airtame : App size: 268M	/Volumes/Airtame 4.13.1/Airtame.app
2025-07-06 22:39:16 : DEBUG : airtame : Debugging enabled, App Verification output was:
/Volumes/Airtame 4.13.1/Airtame.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: AIRTAME APS (4TPSP88HN2)

2025-07-06 22:39:16 : INFO  : airtame : Team ID matching: 4TPSP88HN2 (expected: 4TPSP88HN2 )
2025-07-06 22:39:16 : INFO  : airtame : Installing Airtame version 4.13.1 on versionKey CFBundleShortVersionString.
2025-07-06 22:39:16 : INFO  : airtame : App has LSMinimumSystemVersion: 10.15
2025-07-06 22:39:16 : DEBUG : airtame : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2025-07-06 22:39:16 : INFO  : airtame : Finishing...
2025-07-06 22:39:19 : INFO  : airtame : name: Airtame, appName: Airtame.app
2025-07-06 22:39:19 : WARN  : airtame : No previous app found
2025-07-06 22:39:19 : WARN  : airtame : could not find Airtame.app
2025-07-06 22:39:19 : REQ   : airtame : Installed Airtame, version 4.13.1
2025-07-06 22:39:19 : INFO  : airtame : notifying
2025-07-06 22:39:19 : DEBUG : airtame : Unmounting /Volumes/Airtame 4.13.1
2025-07-06 22:39:19 : DEBUG : airtame : Debugging enabled, Unmounting output was:
"disk4" ejected.
2025-07-06 22:39:19 : DEBUG : airtame : DEBUG mode 1, not reopening anything
2025-07-06 22:39:19 : REQ   : airtame : All done!
2025-07-06 22:39:19 : REQ   : airtame : ################## End Installomator, exit code 0
````

uurazzle reviewed (commented):
Hi @hwlans,

Thank you for the contribution! We noticed the label is missing the `appNewVersion` variable. Including this is highly recommended because it enables Installomator to perform a version check before taking any action.

### Add appNewVersion

The primary benefits are:

- **Avoids unnecessary re-installations:** When `appNewVersion` is present, Installomator compares the latest available version against the version currently installed. If they match, it exits without doing anything. Without it, the app is re-downloaded and re-installed on every script run, even when the user already has the newest version.
- **Improves user experience:** If an app is running, Installomator typically prompts the user to close it before updating. With `appNewVersion`, this prompt only appears when an actual update is available — users won't be asked to quit an app just to reinstall the same version they already have.
- **Enables version pinning:** By overriding `appNewVersion` at runtime, admins can pin devices to a specific version — useful for maintaining compatibility with older macOS versions or holding back a problematic release.
- **Saves bandwidth:** Because the script can exit early when versions match, it prevents unnecessary downloads of large DMG or PKG files, reducing network load across a fleet of managed Macs.

Please have a look at how other labels implement dynamic version detection — common approaches include `versionFromGit`, parsing a Sparkle appcast, or scraping a vendor release page. 

For example here is one possible option:

```
appNewVersion=$(curl -fsIL "https://downloads-website.airtame.com/get.php?platform=mac&pkg=true" | grep -i "^location" | tail -1 | grep -oE "[0-9]+\.[0-9]+\.[0-9]+" | head -1)
```

hwlans committed:
Update airtame.sh

hwlans committed:
Update airtame.sh

Back to type dmg

````
./assemble.sh airtame
2026-05-07 22:31:27 : INFO  : airtame : Total items in argumentsArray: 0
2026-05-07 22:31:27 : INFO  : airtame : argumentsArray:
2026-05-07 22:31:27 : REQ   : airtame : ################## Start Installomator v. 10.9beta, date 2026-05-07
2026-05-07 22:31:27 : INFO  : airtame : ################## Version: 10.9beta
2026-05-07 22:31:27 : INFO  : airtame : ################## Date: 2026-05-07
2026-05-07 22:31:27 : INFO  : airtame : ################## airtame
2026-05-07 22:31:27 : DEBUG : airtame : DEBUG mode 1 enabled.
2026-05-07 22:31:28 : INFO  : airtame : Reading arguments again:
2026-05-07 22:31:28 : DEBUG : airtame : name=Airtame
2026-05-07 22:31:28 : DEBUG : airtame : appName=
2026-05-07 22:31:28 : DEBUG : airtame : type=dmg
2026-05-07 22:31:28 : DEBUG : airtame : archiveName=
2026-05-07 22:31:28 : DEBUG : airtame : downloadURL=https://downloads-website.airtame.com/get.php?platform=mac
2026-05-07 22:31:28 : DEBUG : airtame : curlOptions=
2026-05-07 22:31:28 : DEBUG : airtame : appNewVersion=4.15.0
2026-05-07 22:31:28 : DEBUG : airtame : appCustomVersion function: Not defined
2026-05-07 22:31:28 : DEBUG : airtame : versionKey=CFBundleShortVersionString
2026-05-07 22:31:28 : DEBUG : airtame : packageID=
2026-05-07 22:31:28 : DEBUG : airtame : pkgName=
2026-05-07 22:31:28 : DEBUG : airtame : choiceChangesXML=
2026-05-07 22:31:28 : DEBUG : airtame : expectedTeamID=4TPSP88HN2
2026-05-07 22:31:28 : DEBUG : airtame : blockingProcesses=
2026-05-07 22:31:28 : DEBUG : airtame : installerTool=
2026-05-07 22:31:28 : DEBUG : airtame : CLIInstaller=
2026-05-07 22:31:28 : DEBUG : airtame : CLIArguments=
2026-05-07 22:31:28 : DEBUG : airtame : updateTool=
2026-05-07 22:31:28 : DEBUG : airtame : updateToolArguments=
2026-05-07 22:31:28 : DEBUG : airtame : updateToolRunAsCurrentUser=
2026-05-07 22:31:28 : INFO  : airtame : BLOCKING_PROCESS_ACTION=tell_user
2026-05-07 22:31:28 : INFO  : airtame : NOTIFY=success
2026-05-07 22:31:28 : INFO  : airtame : LOGGING=DEBUG
2026-05-07 22:31:28 : INFO  : airtame : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2026-05-07 22:31:28 : INFO  : airtame : Label type: dmg
2026-05-07 22:31:28 : INFO  : airtame : archiveName: Airtame.dmg
2026-05-07 22:31:28 : INFO  : airtame : no blocking processes defined, using Airtame as default
2026-05-07 22:31:28 : DEBUG : airtame : Changing directory to /Users/h.lans/Developer/GitHub/Installomator/build
2026-05-07 22:31:28 : INFO  : airtame : name: Airtame, appName: Airtame.app
2026-05-07 22:31:28 : WARN  : airtame : No previous app found
2026-05-07 22:31:28 : WARN  : airtame : could not find Airtame.app
2026-05-07 22:31:28 : INFO  : airtame : appversion:
2026-05-07 22:31:28 : INFO  : airtame : Latest version of Airtame is 4.15.0
2026-05-07 22:31:28 : REQ   : airtame : Downloading https://downloads-website.airtame.com/get.php?platform=mac to Airtame.dmg
2026-05-07 22:31:28 : DEBUG : airtame : No Dialog connection, just download
2026-05-07 22:31:33 : INFO  : airtame : Downloaded Airtame.dmg – Type is  zlib compressed data – SHA is 1b19d29cff16d2ef424ee6914f9b2f63b35238fe – Size is 131548 kB
2026-05-07 22:31:33 : DEBUG : airtame : DEBUG mode 1, not checking for blocking processes
2026-05-07 22:31:33 : REQ   : airtame : Installing Airtame
2026-05-07 22:31:33 : INFO  : airtame : Mounting /Users/h.lans/Developer/GitHub/Installomator/build/Airtame.dmg
2026-05-07 22:31:36 : DEBUG : airtame : Debugging enabled, dmgmount output was:
Checksumming Protective Master Boot Record (MBR : 0)…
Protective Master Boot Record (MBR :: verified CRC32 $23B1793C
Checksumming GPT Header (Primary GPT Header : 1)…
GPT Header (Primary GPT Header : 1): verified CRC32 $CB4C920F
Checksumming GPT Partition Data (Primary GPT Table : 2)…
GPT Partition Data (Primary GPT Tabl: verified CRC32 $CC2E9892
Checksumming  (Apple_Free : 3)…
(Apple_Free : 3): verified CRC32 $00000000
Checksumming disk image (Apple_HFS : 4)…
disk image (Apple_HFS : 4): verified CRC32 $6653A6A7
Checksumming  (Apple_Free : 5)…
(Apple_Free : 5): verified CRC32 $00000000
Checksumming GPT Partition Data (Backup GPT Table : 6)…
GPT Partition Data (Backup GPT Table: verified CRC32 $CC2E9892
Checksumming GPT Header (Backup GPT Header : 7)…
GPT Header (Backup GPT Header : 7): verified CRC32 $AD4AC577
verified CRC32 $03C165A0
/dev/disk8          	GUID_partition_scheme
/dev/disk8s1        	Apple_HFS                      	/Volumes/Airtame 4.15.0

2026-05-07 22:31:36 : INFO  : airtame : Mounted: /Volumes/Airtame 4.15.0
2026-05-07 22:31:36 : INFO  : airtame : Verifying: /Volumes/Airtame 4.15.0/Airtame.app
2026-05-07 22:31:36 : DEBUG : airtame : App size: 268M	/Volumes/Airtame 4.15.0/Airtame.app
2026-05-07 22:31:37 : DEBUG : airtame : Debugging enabled, App Verification output was:
/Volumes/Airtame 4.15.0/Airtame.app: accepted
source=Notarized Developer ID
origin=Developer ID Application: AIRTAME APS (4TPSP88HN2)

2026-05-07 22:31:37 : INFO  : airtame : Team ID matching: 4TPSP88HN2 (expected: 4TPSP88HN2 )
2026-05-07 22:31:37 : INFO  : airtame : Installing Airtame version 4.15.0 on versionKey CFBundleShortVersionString.
2026-05-07 22:31:37 : INFO  : airtame : App has LSMinimumSystemVersion: 10.15
2026-05-07 22:31:37 : DEBUG : airtame : DEBUG mode 1 enabled, skipping remove, copy and chown steps
2026-05-07 22:31:37 : INFO  : airtame : Finishing...
2026-05-07 22:31:40 : INFO  : airtame : name: Airtame, appName: Airtame.app
2026-05-07 22:31:41 : WARN  : airtame : No previous app found
2026-05-07 22:31:41 : WARN  : airtame : could not find Airtame.app
2026-05-07 22:31:41 : REQ   : airtame : Installed Airtame, version 4.15.0
2026-05-07 22:31:41 : INFO  : airtame : notifying
2026-05-07 22:31:41 : DEBUG : airtame : Unmounting /Volumes/Airtame 4.15.0
2026-05-07 22:31:41 : DEBUG : airtame : Debugging enabled, Unmounting output was:
"disk8" ejected.
2026-05-07 22:31:41 : DEBUG : airtame : DEBUG mode 1, not reopening anything
2026-05-07 22:31:41 : REQ   : airtame : All done!
2026-05-07 22:31:41 : REQ   : airtame : ################## End Installomator, exit code 0
````

uurazzle reviewed (approved):
Hi @hwlans,

Thanks for contributing this Installomator label ([#2471: Update airtame.sh](https://github.com/Installomator/Installomator/pull/2471))—everything looks good. Appreciate you taking the time to add it to the project!

gilburns reviewed (approved):
Thank you for your contribution to the project!
